frontend/lib/src/components/elements/DeckGlJsonChart/useDeckGl.tsx

```diff
@@ -296,18 +296,33 @@ export const useDeckGl = (props: UseDeckGlProps): UseDeckGlShape => {
 
           const shouldUseOriginalFillFunction = !anyLayersHaveSelection
 
-          if (shouldUseOriginalFillFunction) {
+          const originalFillFunction = layer[fillFunction] as
+            | FillFunction
+            | undefined
+
+          if (shouldUseOriginalFillFunction || !originalFillFunction) {
             // If we aren't changing the fill color, we don't need to change the fillFunction
             return clonedLayer
           }
 
-          const originalFillFunction = layer[fillFunction] as
-            | FillFunction
-            | undefined
+          const selectedOpacity = 255
+          const unselectedOpacity = Math.floor(255 * 0.4)
```

Comment:
what was going on regarding this prior to this change?
> For fallback colors, ensures their opacities are actually set in the expected range.

Reply:
`parseToRgba` returns opacity in a scale of 0 to 1 for some reason. So it effectively always looked completely transparent 😢

Reply:
oh interesting, good to know!